compiler-rt/lib/ubsan/ubsan_handlers.cpp

```diff
@@ -555,13 +555,11 @@ static void handleImplicitConversion(ImplicitConversionData *Data,
                                      ReportOptions Opts, ValueHandle Src,
                                      ValueHandle Dst) {
   SourceLocation Loc = Data->Loc.acquire();
-  ErrorType ET = ErrorType::GenericUB;
-
   const TypeDescriptor &SrcTy = Data->FromType;
   const TypeDescriptor &DstTy = Data->ToType;
-
   bool SrcSigned = SrcTy.isSignedIntegerTy();
   bool DstSigned = DstTy.isSignedIntegerTy();
+  ErrorType ET = ErrorType::GenericUB;
```

Comment:
This seems suspect, given that this handler is detecting things that aren't UB.

Reply:
I have only moved this line... I think changing the error type is for a future PR as you comment further down.

Reply:
Ah, OK. That makes sense.